src/components/KubeConfig/KubeInstallation.vue

```diff
@@ -64,16 +53,11 @@ export default {
 </script>
 
 <style scoped lang="scss">
-  .btn {
-    margin: 2vw 0vw 1vw 0vw;
-    color: floralwhite;
-    background-color: #689F38 !important;
-  }
-  .btn:hover {
-    background-color: #8BC34A !important;
-  }
-  .icon {
-    color: #F1F8E9;
+  .accordion{
+    width: 60vw;
+    margin: 0 auto;
+    //$expansion-panel-border-radius: 0px !important
```

Comment:
This is the potential fix I was talking about, but it doesn't work. I've also added a link to where vuetify explains custom variables.